src/sentry/issues/grouptype.py

```diff
@@ -110,6 +110,21 @@ def expiry_seconds(self) -> int:
         return int(self.expiry_time.total_seconds())
 
 
+@dataclass(frozen=True)
+class NotificationConfig:
+    default_tags: list[str] = field(
+        default_factory=lambda: ["level", "release", "handled", "environment"]
+    )  # TODO(cathy): no tags for crons (empty list)
+    text_code_formatted: bool = True  # TODO(cathy): user feedback wants it formatted as text
+    context: list[str] = field(
+        default_factory=lambda: ["Events", "Users Affected", "State", "First Seen"]
+    )  # see SUPPORTED_CONTEXT_DATA for all possible values
+    actions: list[str] = field(default_factory=lambda: ["archive", "resolve", "assign"])
```

Comment:
For all the lists, should they be sets instead to help prevent duplicates? Or do we want duplicates in some?

Reply:
this is a work in progress but at least for the context, i believe order matters because it's how we're appending to the context string